src/index.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Montserrat;
+  src: url(./components/assets/fonts/Montserrat-Regular.ttf);
+}
+@font-face {
+  font-family: Pacifico;
+  src: url(./components/assets/fonts/Pacifico-Regular.ttf);
+}
+@font-face {
+  font-family: Axis;
+  src: url(./components/assets/fonts/Axis.woff2);
+}
+:root {
+  --pacifico: Pacifico, "Times New Roman", Times, serif;
+  --montserrat: Montserrat, Arial, Helvetica, sans-serif;
+  --axis: Axis, Arial, Helvetica, sans-serif;
+  --black: #000;
+  --white: #fff;
+  --subheading-color: #985e5b;
+  --bg-color-contact: #edeff2;
+  --bg-color-gallery: #f7f7f7;
+  --hover: #d3d3d3;
+}
+.html,
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+.about {
```

Comment:
also I would suggest creating separate stylesheets for each component.

Reply:
Done